### ISSUE TITLE
The command line option `--ext` doesn't work when the filename is specified explicitly.

### ISSUE THREAD

**Issue by Alex1990** [eslint/eslint#5911]
The default value of `--ext` is `.js`. But, when I run the below command, eslint still lints the file. Is it intentional?

```shell
eslint package.json
```

Even I explicitly specify the `--ext` option:

```shell
eslint --ext .js package.json
```

ESLint: v2.8.0
Node: v5.5
OS: Darwin Kernel Version 15.0.0

Also, I find this behavior when I want a Bash variable to represent the filenames. Example:

```shell
files=$(git status  -uno --porcelain | awk 'BEGIN { FS=" "; } { print $2 }' | tr '\n' ' ')
eslint $files
```

**Comment by alberto** [comment]
Yes, explicitly passed files should always be linted by design.
